UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminGlobalAudio.cs

```diff
@@ -12,35 +16,111 @@ namespace AdminTools
 	public class AdminGlobalAudio : MonoBehaviour
 	{
 		[SerializeField] private GameObject buttonTemplate = null;
+		[SerializeField] private Scrollbar loadingBar = null;
+		[SerializeField] private GameObject loadingView = null;
 		private AdminGlobalAudioSearchBar SearchBar;
 		public List<GameObject> audioButtons = new List<GameObject>();
 
 		[SerializeField] private AudioClipsArray audioAddressables = null;
 
-		public List<AddressableAudioSource> audioList;
+		public Dictionary<AddressableAudioSource, string> audioList = new Dictionary<AddressableAudioSource, string>();
+		public static CatalogueData catalogueData;
 
 		private void Awake()
 		{
 			SearchBar = GetComponentInChildren<AdminGlobalAudioSearchBar>();
-			
-			DoLoadAudio(audioAddressables);
+
+			Refresh();
+			DoLoadAudio(catalogueData);
+		}
+
+		public static List<string> GetCataloguePath()
+		{
+			var path = Application.dataPath.Remove(Application.dataPath.IndexOf("/Assets"));
+			path += "/AddressablePackingProjects";
```

Comment:
isn't this only for the editor?

Reply:
let me make a build and check

Reply:
![image](https://github.com/unitystation/unitystation/assets/34368774/e24cd331-d7ee-478f-8e6c-9c575e56e8aa)

Reply:
I fixed it